### ISSUE TITLE
color adjustment scroll wheel requires multiple scrolls

### ISSUE THREAD

**Issue by ngreenwald** [vanvalenlab/deepcell-label#115]
To adjust the colors in RGB mode, the scrollwheel allows the user to increase or decrease the brightness. However, if you move the scrollwheel forward a single click, nothing changes. It looks like the "scrollwheel" doesn't activate unless multiple scrolls are input, which makes it tricky to adjust MIBI/IMC data which has small quantized values.

**Comment by geneva-miller** [comment]
Could you elaborate? When I use the scroll adjustment on an RGB image, I do see the adjustment change after just one scrollwheel increment. Is this happening in a particular mode, on particular images, etc?

**Comment by ngreenwald** [comment]
Hmm, interesting. When I scroll on this NPZ, nothing changes if I only move a single scroll wheel forward. I can even do this 5 or 6 times, spaced out a second apart, and nothing happens. Only once I do multiple scrolls at a time. Seems to be consistent across pixel and label mode
[IMC_PointJ02_row_4_col_7.npz.zip](https://github.com/vanvalenlab/caliban/files/4336167/IMC_PointJ02_row_4_col_7.npz.zip)

**Comment by ngreenwald** [comment]
I think I remember seeing somewhere that the scroll has a minimum value that brightness can be adjusted to. Is there a reason for this? I can't tell if it's because of that or something else, but the scrolling feels a little laggy in terms of adjustments. Also, the minimum value means I can't set the brightness low enough so that the autoflouresence disspears from the image. For example, see attached. 
[BRCA_Point23_row_0_col_0_save_version_0.npz.zip](https://github.com/vanvalenlab/caliban/files/4382262/BRCA_Point23_row_0_col_0_save_version_0.npz.zip)

**Comment by geneva-miller** [comment]
Yeah, I think that reason is basically "trying to get the rescaling to work nicely." I think this ties into the work you did to look into how the channels should be displayed. It's worth revisiting how the rescaling/adjustment plays into it. I'll try and take a look this afternoon.

One of the changes in single-channel display is that shift+scroll adjusts vmin of the display, and scrolling adjusts vmax. That approach might work here.

**Comment by ngreenwald** [comment]
Got it. This is definitely lower priority, just aesthetics.

Thanks!
